embedded/imu_driver/imu_balsaq.hh

```diff
@@ -29,10 +29,11 @@ class ImuBq : public BalsaQ {
   void shutdown();
 
   // Every millisecond
-  const static uint loop_delay_microseconds = 1000;
+  const static uint loop_delay_microseconds = 0;
```

Comment:
Did Ben say this was ok?

This might consume all CPU

Reply:
I am onboard with this, but I'd just check that the rest of the stack seems to run ok.

Reply:
I'm good with it, since the loop blocks on I/O for a bit. If this becomes a CPU hog, we'll have to reevaluate.